### ISSUE TITLE
Is search command working

### ISSUE THREAD

**Issue by OrionRandD** [joshpetit/BibleHubScrapper#13]
It seems that the "search" command is not working :(

**Comment by joshpetit** [comment]
Ah I just took a look it seems like it isn't working. It may have to do with biblehub changing their UI. This package uses webscrapping to do everything and I never expected biblehub to make significant ui changes but I suppose they did that or change the path for searching.

If I get time and a spur to work on this I would likely start with something fresh (rework the entire code base to be more maintainable). I'm glad the query command still works though! Hope that can help you in your study till it's fixed

**Comment by OrionRandD** [comment]
Yeah! It is really useful. I would like also to see a feature for printing
Hebrew and Greek Bibles in the terminal, but I do not know that it is
possible.
There are two interesting projects out there as well you could have a look
at and explore a bit - if you have time: blueletterbible and stepbible...
